ext/standard/array.c

```diff
@@ -6435,13 +6435,11 @@ PHP_FUNCTION(array_combine)
 			} else if (Z_TYPE(values->arData[pos_values].val) != IS_UNDEF) {
 				entry_values = &values->arData[pos_values].val;
 				if (Z_TYPE_P(entry_keys) == IS_LONG) {
-					entry_values = zend_hash_index_update(Z_ARRVAL_P(return_value),
-						Z_LVAL_P(entry_keys), entry_values);
+					entry_values = zend_hash_index_update(Z_ARRVAL_P(return_value), Z_LVAL_P(entry_keys), entry_values);
```

Comment:
pretty sure this whitespace change (plus the one on line 6442) is unintentional? Nothing else changed so it pollutes the diff

Reply:
Removed whitespace changes in commit